multichain-testing/tools/e2e-tools.js

```diff
@@ -164,6 +164,18 @@ export const provisionSmartWallet = async (
   },
 ) => {
   trace('provisionSmartWallet', address);
+  /**
+   * @returns {Promise<{
+   *   balances: Coins;
+   *   pagination: unknown;
+   * }>}
+   *
+   * @typedef {{ denom: string; amount: string }[]} Coins
+   */
+  const getCosmosBalances = () =>
+    lcd.getJSON(`/cosmos/bank/v1beta1/balances/${address}`);
+  progress('before whale', await getCosmosBalances());
```

Comment:
Maybe "before send from whale" so this isn't confused as the whale's balances

Reply:
better but not worth another CI run. this is temporary to debug.